pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1763,6 +1826,7 @@ type ObjectUserQuotaSpec struct {
 	MaxObjects *int64 `json:"maxObjects,omitempty"`
 }
 
+// CephObjectRealm represents a Ceph Object Store Gateway Realm
```

Comment:
Why is this line duplicated here?

Reply:
That looks like a rebase mess up. Will fix.

Reply:
b12efe74a9